### DISCUSSION TITLE
Other Text to Speech Options (Eleven Labs / PlayHT)

### DISCUSSION THREAD

**Discussion by lhyphendixon** [langgenius/dify#5450] [Suggestion]
Hey everyone, 

I absolutely love Dify.  It's definitely a game-changer.  One of the key things missing for my needs is the ability to integrate other Text to Speech options like Eleven Labs or PlayHT.  Any chance that can be on the roadmap?

**Comment by lhyphendixon** [comment]
I find it so surprising that I seem to be the only one that doesn't want to use Open AI's voices... I'm willing to pay to have this implemented.  Any devs want to quote me?

**Reply by xiangweizheng** [reply]
me, too. the platform deserve more voice

**Comment by rehberim360** [comment]
Hey everyone,

I absolutely agree with the need for more Text to Speech options! I would also like to suggest adding [Piper](https://github.com/rhasspy/piper) as a TTS option. Piper supports multiple languages and comes with pre-trained models, making it highly versatile. One of the key advantages is that it can be used locally, which provides more control and privacy, especially for those who prefer not to rely on cloud-based solutions. I believe Piper could be a strong addition alongside options like Eleven Labs or PlayHT.

Thanks for considering this!
